cmd/plural/deploy.go

```diff
@@ -386,7 +386,15 @@ func (p *Plural) destroy(c *cli.Context) error {
 			return api.GetErrorResponse(err, "GetInstallation")
 		}
 
-		return p.doDestroy(repoRoot, installation)
+		if err := p.doDestroy(repoRoot, installation); err != nil {
+			return err
+		}
+		if confirm(fmt.Sprintf("Would you like to remove %s from app.plural.sh?", repoName)) {
+			if err := p.ResetInstallation(installation.Id); err != nil {
```

Comment:
I actually have a change that does this as well that might be a bit more in line with what I'd expect the right tear-down is, could we perhaps rip out the uninstall code and limit this to just calling destroyCluster at the end